node/repo/imports/manager.go

```diff
@@ -107,6 +108,7 @@ type Meta struct {
 // CreateImport initializes a new import, returning its ID and optionally a
 // CAR path where to place the data, if requested.
 func (m *Manager) CreateImport() (id ID, err error) {
+	ctx := context.TODO()
```

Comment:
Probably open a tracking issue about this (in markets?)

Reply:
Not sure what it should say and where it should be; what's the problem here?

Reply:
IIRC This `Manager` struct implements some interface in markets, it would be nice to have contexts plumbed, but I don't think it needs to happen right now - so we could just open an issue saying 'add contexts to this stuff'

Reply:
opened #7813.